src/incubator/TextField/FloatingPlaceholder.tsx

```diff
@@ -52,14 +54,17 @@ const FloatingPlaceholder = ({
           scale: interpolateValue(animation, [1, FLOATING_PLACEHOLDER_SCALE])
         },
         {
-          translateX: interpolateValue(animation, [0, -placeholderOffset.left])
+          translateX: interpolateValue(animation, [
+            0,
+            -placeholderOffset.left - extraOffset / FLOATING_PLACEHOLDER_SCALE
```

Comment:
Not sure why but the whole animation is not working when changing the `FLOATING_PLACEHOLDER_SCALE` to another value (unrelated to this PR).

Reply:
We'll need to take a look at it some other time. 
If it'll raise we will investigate it